### ISSUE TITLE
import cv2.aruco as aruco ModuleNotFoundError: No module named 'cv2.aruco'

### ISSUE THREAD

**Issue by thompson318** [UCL/scikit-surgerybard#25]
In GitLab by @MianAhmad on Jul 3, 2019, 09:59

I found a bug in the package which is noticed on some computers. When we run

```
python sksurgerybard.py --config config/reference_only.json
```
We get the following error.     

```
import cv2.aruco as aruco ModuleNotFoundError: No module named 'cv2.aruco'
```

**Comment by thompson318** [comment]
In GitLab by @MianAhmad on Jul 3, 2019, 10:26

mentioned in merge request !5

**Comment by thompson318** [comment]
In GitLab by @MattClarkson on Jul 3, 2019, 11:06

closed via commit 6b926388b7a46515b07b3f78a16e0e5cac9f12c3
